cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -165,22 +167,22 @@ function reconcile_addons() {
   # Filter out `configured` message to not noisily log.
   # `created`, `pruned` and errors will be logged.
   log INFO "== Reconciling with deprecated label =="
-  ${KUBECTL} ${KUBECTL_OPTS} apply -f ${ADDON_PATH} \
+  "${KUBECTL}" "${KUBECTL_OPTS[@]}" apply -f "${ADDON_PATH}" \
```

Comment:
Are you sure KUBECTL_OPTS is an array? I suspect it isn't?

Reply:
Yes, It should be a string.
But I've converted it in Line30:
```sh
IFS=" " read -r -a KUBECTL_OPTS <<< "${KUBECTL_OPTS:-}"
```

Reply:
let's not convert it if we are not converting it EVERYWHERE, including the declaration